packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -97,3 +99,11 @@ String debugFormatDouble(double value) {
   }
   return value.toStringAsFixed(1);
 }
+
+/// A setting that can be used to override the platform [Brightness] exposed
+/// from [BindingBase.window].
+///
+/// See also:
+///
+///  * [WidgetsApp], which uses the [debugBrightnessOverride] setting to construct a [MediaQueryData].
```

Comment:
maybe add that it only uses it in debug mode.

Reply:
Done